McAfee/mcafee_web_gateway/tests/mcafee_access_log.json

```diff
@@ -0,0 +1,43 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "mcafee-web-gateway",
+        "dialect_uuid": "40bac399-2d8e-40e3-af3b-f73a622c9687"
+      }
+    },
+    "message": "[06/Feb/2020:14:24:53 +0100] \"(BypassAuthentication-UserAgent)\" 1.2.3.4 403 \"GET http://ctldl.windowsupdate.com/msdownload/update/v3/static/trustedr/en/disallowedcertstl.cab?2975230343cffee9 HTTP/1.1\" \"Software/Hardware\" \"Minimal Risk\" \"\" 3252 238 \"Microsoft-CryptoAPI/10.0\" \"\" \"200\" \"\""
```

Comment:
This test case fails on `['event']['action']` field added when running locally

Reply:
fixed